werkzeug/http.py

```diff
@@ -558,15 +558,24 @@ def parse_range_header(value, make_inclusive=True):
         if item.startswith('-'):
             if last_end < 0:
                 return None
-            begin = int(item)
+            try:
+                begin = int(item)
+            except ValueError:
+                return None
             end = None
             last_end = -1
         elif '-' in item:
             begin, end = item.split('-', 1)
+            begin = begin.strip()
+            end = end.strip()
+            if not begin.isdigit():
```

Comment:
Using `isdigit` just before allows to drop things like `+11-12`.
And if it passes the `isdigit` test, `int` will never fail, so no need for try-except.